### PR TITLE
Feature project skeleton - issue number #5

### PR DESCRIPTION
**Pull request by cawak**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 var express = require("express");
-var http = require("http");
 var app = express();
+var dao = require('./app/dao/dao');
+var port = process.env.PORT || 8000;
```
Comment (segalmc) [inline index.js:4]: what is process.env.PORT? we dont have jenkins

File: index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+require('./app/routes/routes')(app, dao);
```
Comment (segalmc) [inline index.js:5]: I don't understand what (app, dao) this mean?

File: index.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
 var express = require("express");
-var http = require("http");
 var app = express();
+var dao = require('./app/dao/dao');
```
Comment (segalmc) [inline index.js:3]: way to layer dao?

File: package.json [hunk 4]
```diff
@@ -2,10 +2,12 @@
   "name": "pooper",
   "author": "gamasak",
   "private": true,
+  "main": "pooper",
   "scripts": {
     "start": "node index.js"
   },
   "dependencies": {
     "express": "^4.16.2"
```
Comment (segalmc) [inline package.json:11]: please specify specific version

**Review by segalmc — APPROVED** [review]

File: index.js (hunk 3 above)
Comment (cawak) [reply index.js:3]: Yes, I thought that we will use a reference to the dao instead of a reference to a specific db client.

File: index.js (hunk 2 above)
Comment (cawak) [reply index.js:5]: I need to pass an instance of express and dao to populate it with routings and to use the dao.

File: package.json (hunk 4 above)
Comment (cawak) [reply package.json:11]: hehe. What version do you think is suitable?

File: index.js (hunk 1 above)
Comment (cawak) [reply index.js:4]: Heroku adds the allowed port into this env variable.

**Pull request merged**